HAL/pico/src/comms/DInputBackend.cpp

```diff
@@ -33,27 +33,31 @@ void DInputBackend::SendReport() {
     UpdateOutputs();
 
     // Digital outputs
-    _gamepad->setButton(0, _outputs.b);
-    _gamepad->setButton(1, _outputs.a);
-    _gamepad->setButton(2, _outputs.y);
-    _gamepad->setButton(3, _outputs.x);
-    _gamepad->setButton(4, _outputs.buttonR);
-    _gamepad->setButton(5, _outputs.triggerRDigital);
-    _gamepad->setButton(6, _outputs.buttonL);
-    _gamepad->setButton(7, _outputs.triggerLDigital);
-    _gamepad->setButton(8, _outputs.select);
-    _gamepad->setButton(9, _outputs.start);
-    _gamepad->setButton(10, _outputs.rightStickClick);
-    _gamepad->setButton(11, _outputs.leftStickClick);
-    _gamepad->setButton(12, _outputs.home);
+    // See https://wiki.libsdl.org/SDL2/SDL_GameControllerButton
+    _gamepad->setButton(0, _outputs.a);
+    _gamepad->setButton(1, _outputs.b);
+    _gamepad->setButton(2, _outputs.x);
+    _gamepad->setButton(3, _outputs.y);
+    _gamepad->setButton(4, _outputs.select);
+    _gamepad->setButton(5, _outputs.home);
+    _gamepad->setButton(6, _outputs.start);
+    _gamepad->setButton(7, _outputs.leftStickClick);
+    _gamepad->setButton(8, _outputs.rightStickClick);
+    _gamepad->setButton(9, _outputs.buttonL);
+    _gamepad->setButton(10, _outputs.buttonR);
+    _gamepad->setButton(11, _outputs.triggerLDigital);
+    _gamepad->setButton(12, _outputs.triggerRDigital);
```

Comment:
Unnecessary, but helped when using DInput in my game engine. Doesn't really affect anything.